pmoves/services/session-context-worker/main.py

```
@@ -65,7 +66,41 @@
+
+    if _nc:
+        try:
+            await _nc.close()
+        except Exception:
+            pass
+        _nc = None
+
+
+if __name__ == "__main__":
+    import uvicorn
+    uvicorn.run(app, host="0.0.0.0", port=HEALTH_PORT)
```

Ruff (0.14.10)

Undefined name `app`

(F821)